SPECS/qemu.spec

```
@@ -198,6 +201,10 @@ cp -r scripts/qmp %{buildroot}%{_datarootdir}/qemu
 %{?_cov_results_package}
 
 %changelog
+* Tue Feb 04 2025 Lucas Ravagnier <lucas.ravagnier@vates.tech> - 4.2.1-4.6.4.2
+- Add qemu-4.2.1-CVE-2023-3354.backport.patch to fix a CVE
+of a NULL pointer dereference that can cause a DOS
```

Please align the new line:
```suggestion
  of a NULL pointer dereference that can cause a DOS
```